app/views/users/show.html.erb

```diff
@@ -1,4 +1,7 @@
-<h2><%= @user.username %></h2>
+<div class='user-crawler'>
+	<img src='<%= get_absolute_image_url(@user.small_avatar_url) %>' alt='Image of <%= @user.username %>' title='<%= @user.username %>' />
```

Comment:
The alt should be localized

Reply:
localization and me :bowing_man:, thanks @ZogStriP, I have fixed it now